lib/Pool.js

```diff
@@ -244,6 +248,7 @@ class Pool {
   // odbc.connect runs on an AsyncWorker, so this is truly non-blocking
   async increasePoolSize(count) {
     this.connectingCount += count;
+    this.poolSize += count;
```

Comment:
Instead of incrementing here by the entire count, should maybe increment one at a time after a successful call to `odbc.connect` and right before the line `connection = new Connection(nativeConnection);`

Reply:
Good point---I'm assuming all the connections will succeed. I'll make the change.